includes/object-cache.php

```diff
@@ -2138,6 +2242,50 @@ protected function handle_exception( $exception ) {
     public function __get( $name ) {
         return isset( $this->{$name} ) ? $this->{$name} : null;
     }
+
+    /**
+     * Calculates how much time has elapsed since $start_time.
+     *
+     * @param float $start_time
+     * @return float
+     */
+    private function elapsed( $start_time ) {
+        return microtime(true) - $start_time;
```

Comment:
If `microtime()` isn't good enough, we could use `hrtime(true) * 1e-9`.

Reply:
`hrtime` is only available since PHP 7.3 so we should not use it for now but it might be a good feature addition candidate for version 3